SPECS-EXTENDED/libunicap/libunicap.spec

```
@@ -0,0 +1,243 @@
+Summary:        Library to access different kinds of (video) capture devices
+Name:           libunicap
+Version:        0.9.12
+Release:        28%{?dist}
+License:        GPLv2+
```

Nit: Could be an SPDX expression

```suggestion
License:        GPL-2.0-or-later
```